travis/Dockerfile-travis-web

```diff
@@ -13,5 +13,6 @@ WORKDIR /src
 COPY . /src
 
 RUN chown -R mitodl:mitodl /src
+RUN mkdir /var/media && chown -R mitodl:mitodl /var/media
```

Comment:
Do we need to do this here? the `mm_web_travis` container that this inherits from is built using the root-level `Dockerfile`.